index.bs

```diff
@@ -1309,6 +1345,7 @@ The {{MLGraphBuilder}}.{{MLGraphBuilder/build()}} method compiles the graph buil
     The [=new=] <dfn constructor for=MLGraphBuilder lt="MLGraphBuilder(context)">MLGraphBuilder(|context|)</dfn> constructor steps are:
   </summary>
     1. If [=this=]'s [=relevant global object=]'s [=associated Document=] is not [=allowed to use=] the [=webnn-feature|webnn=] feature, then [=exception/throw=] a "{{SecurityError}}" {{DOMException}}.
+    1. If |context| [=MLContext/is lost=], then [=exception/throw=] an "{{InvalidStateError}}" {{DOMException}}.
```

Comment:
For completeness this check should be attached to basically every method, but in particular the ones that require interaction with the hardware such as `build()`, `compute()` and `dispatch()`.

It's an interesting question whether if we implement the optimization @a-sully is working on for `constant()` we should have that method throw if the context is lost or just ignore the invalid state until `build()` is called.

Reply:
Added to `build()` in 01cf420; already present in `compute()`, and `dispatch()` isn't in the spec yet.

We maybe want to change the "If this.[[hasBuilt]] is true, then ..." tests to be "If this **can build**, then..." and define **can build** for a builder as ![[has built] && context is not lost as  Thoughts?

Reply:
Oh, I like that idea!

Reply:
Okay, tried it out in d2ec3f4